packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -105,6 +117,21 @@ export async function handle(args: Result<ReturnType<typeof options>>) {
     process.exit(1)
   }
 
+  // If the user passes `{bin} build --map -` then this likely means they want to output the map inline
+  // this is the default behavior of `{bin build} --map` to inform the user of that
+  if (args['--map'] === '-') {
+    eprintln(header())
+    eprintln()
```

Comment:
This recently got hoisted, so I would suggest to delete this (and rebase on `main`)

Reply:
done